src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -64,13 +64,13 @@ public class SignInManager<TUser> where TUser : class
         {
             if (Options.SignIn.RequireConfirmedEmail && !(await UserManager.IsEmailConfirmedAsync(user)))
             {
-                return await LogResultAsync(false, user);
+                return await false.WithLoggingAsync(this, user);
             }
             if (Options.SignIn.RequireConfirmedPhoneNumber && !(await UserManager.IsPhoneNumberConfirmedAsync(user)))
             {
-                return await LogResultAsync(false, user);
+                return await false.WithLoggingAsync(this, user);
```

Comment:
Personally I think this looks kind of ugly compared to the earlier iteration, extension off of false just seems really strange, also now we are passing this everywhere, which makes me think the methods really belong on the appropriate Manager